src/config/rcxml.c

```diff
@@ -1465,10 +1467,14 @@ post_processing(void)
 
 	int nr_workspaces = wl_list_length(&rc.workspace_config.workspaces);
 	if (nr_workspaces < rc.workspace_config.min_nr_workspaces) {
+		char *ws_name = xstrdup("Workspace");
+		if (rc.workspace_config.prefix) {
+			ws_name = rc.workspace_config.prefix;
+		}
```

Comment:
Suggest something like this instead
```c
if (!rc.workspace_config.prefix) {
	rc.workspace_config.prefix = xstrdup("Workspace");
}
[..]
workspace->name = strdup_printf("%s %d", rc.workspace_config.prefix, i + 1);
```

This also needs a `zfree(rc.workspace_config.prefix)` in `rcxml_finish()`, otherwise we will leak memory.